server/test/test_gdpr_endpoints.py

```
@@ -0,0 +1,123 @@
+            f"http://localhost:{PORT}/api/profiles/gdpr",
+            method="DELETE",
+            headers=headers,
+        )
+        with urllib.request.urlopen(delete_req) as resp:
+            assert resp.getcode() == 200
+
+        try:
+            urllib.request.urlopen(
+                urllib.request.Request(export_url, headers=headers)
+            )
+            assert False
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)